README.md

```diff
@@ -1,24 +1,76 @@
+AAPB has involved the CLAMS team in a collaboration to develop archiving technology for public media (primarily video and audio from publically-funded tv shows and radio broadcasts). This will facilitate the research and preservation of significant historical content from such media.  
+The process of archiving, summarizing and extracting-metadata from this media could eventually be automatic.  
+This repository/endeavor provides training and evaluation data for the Computer Vision and Machine Learning tools in this process.  
+Data collected is used to evaluate the success of the tools. Then, the tools will be trained on larger batches of data to automatically retrieve information important to the archival process.
 
-Each directory in this repository contains a dataset from a specific annotation project. The directory name is the name of the project. Each directory contains the following files:
+## Structure of This Repository/Directory
+ - `batches` subdirectory (1)
+ - project subdirectories (5)
+ - this repository's `readme.md`
 
-* `README.md`: annotation project-specific information, such as
+### Batches Subdirectory 
+The first directory is the special `batches` directory. This special directory maintains tracking information for the whole repository/annotation effort -  
+Batches are the identity tags of a group of media/GUID annotated for a project-found-below in one go. 
+There are possibly multiple times/events where data is annotated/created for this dataset. Each event is a batch.
+Specifically, this directory contains `.txt` files named after the batch name. Batches are usually named after their relevant GitHub issue from [AAPB-CLAMS collaboration repository](https://github.com/clamsproject/aapb-collaboration).  
+  * Each line in a `.txt` file _must_ be a single AAPB GUID, with an exception of any lines starts with `#` which denotes a comment.  
+  * A GUID is a unique identifying string that can be used at their website to find one particular media and its supporting files, eg. `cpb-aacip-96d289b264c` at https://americanarchive.org/catalog/cpb-aacip-96d289b264c.
+
+### Project Subdirectories 
+Each directory in this repository represents a specific annotation project, its datasets and processing tools.    
+This includes its `raw annotated data file`, `software-suite for converting from raw to gold`, `gold-formatted-final-output data file for tool ingestion`,
+and `that project's readme.md explaining it plus its annotation guidelines`.  
+The directory name is the name of the project. Each directory contains the following files:
+
+* **RAW INTERMEDIATE DATASET FILES**: `YYMMDD-batchName` directory - these sub-directories contains raw output files from the manual annotation process created by the annotation tool (or by hand like a `.csv` file). 
+Different annotation tools create different file formats with diverse formatting. 
+  * The `YYMMDD-` prefix _must_ indicate the time when a batch of annotation is conducted. 
```

Comment:
not very clear what "conducted" means. How about "an annotation guideline is established and the batch to annotate is decided" ?

Reply:
This was actually original wording i inherited. 
Do you want more specificity like this?

Reply:
Not sure about the guidelines as they should be evolving over time. Perhaps the date when the batch is decided work. I do want to have a specificity to `DD` level since I want the order of annotation process to be shown somehow just in the file system, instead of finding it by actually opening a readme file and reading though it.